manifest_staging/charts/gatekeeper/crds/expansiontemplate-customresourcedefinition.yaml

```diff
@@ -28,6 +28,10 @@ spec:
             description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
+            properties:
+              name:
+                maxLength: 63
```

Comment:
Can we also add a check in the ET validation code, to handle the shift-left use case?